Buildings/Controls/OBC/CDL/Logical/VariablePulse.mo

```diff
@@ -0,0 +1,329 @@
+within Buildings.Controls.OBC.CDL.Logical;
+block VariablePulse
+  "Generate boolean pulse with the width specified by input"
+
+  parameter Real samplePeriod(unit="s")
+    "Sample period of the width input";
+  parameter Real period(unit="s")
+    "Time for one pulse period";
+  parameter Real chaWidThr=0.05
+    "Threshold for checking if the width input is changed. It is the ratio of the value change to the original value"
+    annotation (Dialog(tab="Advanced"));
+  parameter Real zerWidThr=0.01
```

Comment:
Let's discuss. I think what this means and what you want to accomplish is something like "Minimum value for `u` below which the output remains always `false`.

Reply:
changed to `Minimum value of input below which the output remains always false`.